btp/autoscaler/config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "instance_min_count": 1,
+  "instance_max_count": 5,
+  "scaling_rules": [
+    {
+      "metric_type": "cpuutil",
+      "threshold": 85,
```

Comment:
to be more precise, add `breach_duration_secs ` property to explicitly define how much time this condition must be met until amount of instances is increased: https://help.sap.com/docs/application-autoscaler/application-autoscaler/parameters-for-dynamic-scaling-policy?locale=en-US&q=breach_duration_secs

E.g.:
```
{
      "metric_type": "cpuutil",
      "operator": ">=",
      "threshold": 85,
      "breach_duration_secs": 60,
      "adjustment": "+1"
    },
    {
      "metric_type": "cpuutil",
      "operator": "<",
      "threshold": 85,
      "breach_duration_secs": 60,
      "adjustment": "-1"
    }
```